packages/devtools_app/test/test_infra/flutter_test_environment.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: implementation_imports, import_of_legacy_library_into_null_safe
+// ignore_for_file: implementation_imports
```

Comment:
ditto

Reply:
done in https://github.com/flutter/devtools/pull/4011